src/FSLibrary/StellarCoreCfg.fs

```diff
@@ -247,6 +247,8 @@ type StellarCoreCfg =
         if self.network.missionContext.enableBackgroundSigValidation then
             t.Add("EXPERIMENTAL_BACKGROUND_TX_SIG_VERIFICATION", true) |> ignore
 
+        t.Add("EXPERIMENTAL_TRIGGER_TIMER", true) |> ignore
+
```

Comment:
Should this be guarded behind a flag? Or is it OK for `EXPERIMENTAL_TRIGGER_TIMER` to always be enabled?